rozenborg/test-simulations-mgmt-repo
# Chores(Automate Task, Check repo health, Create repo for project, Setup linters, Check for build violations)

#### What does this PR do?
- This is to merge the working branch to the output branch.

#### Description of Task to be completed?
- Setup Gulp task runner on the project.
- Setup linters (ESLint)
- Check for Travis CI build errors.
- Check for HoundCI comments on style violations
- Demonstrates HTML, CSS and Javascript skills
- Setup Code Climate and Coveralls on repo.


#### How should this be manually tested?
- From repo root directory, run `gulp` to start Gulp task runner
- This opens up the app page.
- All modifications done on the source files (i.e. html, css and js) would reflect on the app page.
- Check screencast (link stated below) for a short demo.
- HoundCI comments are generated for every Pull request
- 

#### Any background context you want to provide?
- TravisCI checks for code integrity for every build.
- HoundCI prompts for every style violations
- Gulp help automate chores e.g. browser refresh, file minification and bundling.

#### Relevant Pivotal Tracker stories
- NA

#### Screenshots
-<img width="1680" alt="screen shot 2017-04-01 at 9 41 20 am" src="https://cloud.githubusercontent.com/assets/26272628/24577087/67ea4b4e-16bf-11e7-84e4-98c3d945a564.png">

#### Screencast
-[Youtube link to screencast](https://youtu.be/kb6hBtPeIOI)
